app/Http/Controllers/Admin/UsersController.php

```diff
@@ -49,43 +46,25 @@ public function delete($userId) {
 
         // Validate the user ID
         if (!User::where('id', $userId)->exists() || $userId == \Auth::user()->id) {
-            $users = User::all();
-            return view('admin/users/overview')->with('users', $users)->with('failure', true);
+            \Session::flash('failure', trans('admin.delete_user_failed'));
```

Comment:
Deze flash kan niet werken als je het doet zoals het moet, kijk even in app.blade.php in resources/layouts.